package.json

```diff
@@ -22,7 +22,7 @@
     "mime-types": "2.1.3",
     "moment": "2.9.0",
     "moment-timezone": "0.4.0",
-    "openstax-react-components": "openstax/react-components#a8a29638c07cdad0d693ac41da4a9957b922878a",
+    "openstax-react-components": "openstax/react-components#f16c5b2d9a6ea1365a92eb0b38ed6ccfa2de70d2",
```

Comment:
In the future can you use the commit for the merged PR in here (in case any other changes were made to `react-components` in the meantime?) I think it doesn't matter too much since nothing else was changed in `react-components`